x509-ext/src/lib.rs

```diff
@@ -0,0 +1,62 @@
+#![cfg_attr(not(any(test, feature = "std")), no_std)]
+#![cfg_attr(docsrs, feature(doc_cfg))]
+#![doc = include_str!("../README.md")]
+#![doc(
+    html_logo_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg",
+    html_favicon_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg",
+    html_root_url = "https://docs.rs/x509-ext/0.1.0"
+)]
+#![forbid(unsafe_code)]
+#![warn(missing_docs, rust_2018_idioms)]
+
+use der::{asn1::ObjectIdentifier, Sequence};
+
+extern crate alloc;
+
+pub mod pkix;
+
+/// A trait expressing the OID associated with an extension.
+pub trait Typed {
+    /// The OID which identifies an extension.
+    const OID: ObjectIdentifier;
+}
```

Comment:
This might be interesting to add to `const-oid`. There are a few other places it could be used.

Reply:
Done.